### PR TITLE
Fixes borg emotes, wooooooooo!

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request
Fixes borg emotes, by making all borg emotes be within synth_emote.dm.

## How This Contributes To The Skyrat Roleplay Experience
Someone messed up the borg emotes when they added synth emotes. This fixes them. Woo.

## Changelog

:cl:
fix: Fixes all borg emotes. Woo! Also adds some new emotes to borgs.
refactor: Borg emotes are now handled in synth_emote.dm.
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
ignore the commit name idk why its that

File: code/datums/emotes.dm [hunk 1]
```diff
@@ -64,6 +64,7 @@
 	//SKYRAT EDIT ADDITION BEGIN - EMOTES
 	var/sound_volume = 25 //Emote volume
 	var/list/allowed_species
+	var/silicon_allowed = FALSE
```
Comment (GoldenAlpharex) [inline code/datums/emotes.dm:68]: Please document the variable, even if it's straight forward.

File: modular_skyrat/modules/emotes/code/silicon_emotes.dm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/datum/emote/silicon
+/* /datum/emote/silicon
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/emotes/code/silicon_emotes.dm]: Reeeemove the file if it's being deleted ;-; I need to make a new contrib update don't I?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by nikothedude** [comment]
*inhale FUCK ci

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**